src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer/index.js

```diff
@@ -1,68 +1,66 @@
-import React from 'react';
+import React, {useCallback, useEffect, useRef} from 'react';
 import _ from 'underscore';
+
+import ControlSelection from '../../../../libs/ControlSelection';
+import * as DeviceCapabilities from '../../../../libs/DeviceCapabilities';
 import withLocalize from '../../../withLocalize';
 import htmlRendererPropTypes from '../htmlRendererPropTypes';
 import BasePreRenderer from './BasePreRenderer';
-import * as DeviceCapabilities from '../../../../libs/DeviceCapabilities';
-import ControlSelection from '../../../../libs/ControlSelection';
 
-class PreRenderer extends React.Component {
-    constructor(props) {
-        super(props);
-
-        this.scrollNode = this.scrollNode.bind(this);
-        this.debouncedIsScrollingVertically = _.debounce(this.isScrollingVertically.bind(this), 100, true);
-    }
-
-    componentDidMount() {
-        if (!this.ref) {
-            return;
-        }
-        this.ref.getScrollableNode().addEventListener('wheel', this.scrollNode);
-    }
-
-    componentWillUnmount() {
-        this.ref.getScrollableNode().removeEventListener('wheel', this.scrollNode);
-    }
+function PreRenderer(props) {
```

Comment:
@teneeto Please use prop destructuring -> https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#destructuring

NOTE: Do not remove `defaultProps`, only use prop destructuring.

Reply:
I think here is just forwarding the props to the `BasePreRenderer`, so maybe, in this case, is not need 🤔

Reply:
I think same too.